app/static/pages/volunteer/tabs_template/design_tab.html

```diff
@@ -15,14 +15,7 @@
       with an international team of UX and graphic designers, and positively
       impact the lives of millions of students.
     </p>
-    <p class="oppia-foundation-bold-text">
-      What we do:
-    </p>
-    <ul>
-      <li>Create graphics for lessons</li>
-      <li>Address UX issues that students face</li>
-      <li>Make it easy for lesson creators to get started</li>
-    </ul>
+    <ng-include src="volunteerWorkDescription['design']"></ng-include>
```

Comment:
volunteerWorkDescriptionUrls.design (use dot notation if the key is a string constant).

Ditto elsewhere.